doc/ref_model/chapters/chapter08.md

```diff
@@ -1,4 +1,320 @@
+Environmentally challenged | Outdoors and/or exposed to environmentally harsh conditions | maybe unprotected<br>Exposure to abnormal levels of noise, vibration, heat, chemical, electromagnetic pollution | May only have battery power<br>Low/Medium bandwidth Fixed and/or mobile network access | No or minimal access control | Expensive ruggedisation<br>Operationally complex | Example locations: curbside, near cellular radios, |
+
+<a name="8.3.2"></a>
+### 8.3.2 Telco Edge Cloud Infrastructure Characteristics
+Commodity hardware is only suited for environmentally friendly environments. Commodity hardware have standardised designs and form factors. Cloud deployments in data centers typically use such commodity hardware with standardised configurations resulting in operational benefits for procurement, installation and ongoing operations.
+
+In addition to the type of infrastructure hosted in data center clouds, facilities with smaller sized infrastructure deployments, such as central offices or co-location facilities, may also host non-standard hardware designs including specialised components. The introduction of specialised hardware and custom configurations increases the cloud operations and management complexity.
+
+At the edge, the infrastructure may further include ruggedised hardware for harsh environments and hardware with different form factors.
+
+<a name="8.3.3"></a>
+### 8.3.3 Telco Edge Cloud Infrastructure Profiles
+The [Cloud Infrastructure Profiles](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#4.2.4) section specifies two infrastructure profiles:
+
+The **Basic** cloud infrastructure profile is intended for use by both IT and Network Function workloads that have low to medium network throughput requirements.
+
+The **Network Intensive** cloud infrastructure profile is intended for use by applications that have high network throughput requirements (up to 50Gbps).
+
+The Network Intensive profile can specify extensions for hardware offloading; please see [Hardware Acceleration Abstraction](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter03.md#3.8). The Reference Model Network Intensive profile includes an initial set of [Network Intensive profile extensions](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#42421-network-acceleration-extensions).
+
+Based on the infrastructure deployed at the edge, the Table 8-3 specifies the [Infrastructure Profile features and requirements](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md) that would need to be relaxed.
+
+**Table 8-3. TEC Exceptions to [Infrastructure Profile features and requirements](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md)**
+
+<table>
```

Comment:
Please do not use raw html tables.

Reply:
@CsatariGergely This table needed to be in HTML because column spans

Reply:
@CsatariGergely replaced html with md - formatting not as good